templates/elasticsearch.yml.j2

```diff
@@ -55,3 +55,22 @@ xpack.notification.email:
         password: {{ es_mail_config['pass'] }}
         {%- endif %}
 {% endif %}
+
+{% if es_enable_http_ssl | bool %}
+xpack.security.http.ssl.enabled: true
+xpack.security.http.ssl.key: "{{ es_ssl_certificate_path }}/{{ es_ssl_key | basename }}"
+xpack.security.http.ssl.certificate: "{{ es_ssl_certificate_path }}/{{ es_ssl_certificate | basename }}"
```

Comment:
Handling P12 certificates in addition of PEM would be great (using `xpack.security.http.ssl.keystore.path` & `xpack.security.http.ssl.truststore.path`).